.vib/matomo/goss/matomo.yaml

```diff
@@ -0,0 +1,55 @@
+group:
+  daemon:
+    exists: true
+user:
+  daemon:
+    exists: true
+file:
+  /opt/bitnami/apache/conf/vhosts/htaccess/matomo-htaccess.conf:
+    exists: false
+  /opt/bitnami/matomo:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: daemon
+    group: root
+  /opt/bitnami/matomo/tmp:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: daemon
+    group: root
+  /opt/bitnami/matomo/misc/:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: daemon
+    group: root
+  /opt/bitnami/matomo/misc/user:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: daemon
+    group: root
+  /opt/bitnami/matomo/plugins:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: daemon
+    group: root
```

Comment:
You can use the `check-directories` goss template to test this, as it now accepts defining `owner/group` for the checked dirs.